flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPoolBatchSlotRequestTest.java

```diff
@@ -276,4 +289,28 @@ private void advanceTimeAndTriggerCheckBatchSlotTimeout(TestingSlotPoolImpl slot
 		// timeout all as unfulfillable marked slots
 		slotPool.triggerCheckBatchSlotTimeout();
 	}
+
+	private TestingSlotPoolImpl createAndSetUpSlotPool(
+			final ComponentMainThreadExecutor componentMainThreadExecutor,
+			final ResourceManagerGateway resourceManagerGateway,
+			final Time batchSlotTimeout) throws Exception {
+
+		return SlotPoolUtils
```

Comment:
can be based on the following `createAndSetUpSlotPool （）`

Reply:
I don't agree on this. If we rebase this `createAndSetUpSlotPool` with the one below, we have to introduce a default value of `Clock`, which is unnecessary.

Reply:
I see. Makes sense.